webapp/javascript/components/TagsBar.tsx

```diff
@@ -102,6 +110,33 @@ function QueryInput({ initialQuery, onSubmit }: QueryInputProps) {
     }
   }, [query]);
 
+  useEffect(() => {
+    setTextAreaSize({
+      width: textareaRef?.current?.['offsetWidth'] || 0,
+      height: textareaRef?.current?.['offsetHeight'] || 0,
+    });
+  }, [query, windowWidth, onSubmit]);
+
+  const onChange = (e: any) => {
```

Comment:
Can we type this any? :)

Reply:
done :)

Reply:
Wait i still see it there

Reply:
some kind of github caching issue
![image](https://user-images.githubusercontent.com/7372044/165330934-485f0179-1ace-440d-bec8-5e00b08098b6.png)